packages/default-theme/cms/sections/CmsSectionSidebar.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="sw-sidebar-section">
+    <div class="sw-sidebar-section__sidebar" v-if="getSidebarBlocks.length">
+      <CmsGenericBlock
+        v-for="cmsBlock in getSidebarBlocks"
+        :key="cmsBlock.id"
+        :content="cmsBlock"
+      />
+    </div>
+    <div class="sw-sidebar-section__main">
+      <CmsGenericBlock
+        v-for="cmsBlock in getMainBlocks"
+        :key="cmsBlock.id"
+        :content="cmsBlock"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import CmsGenericBlock from 'sw-cms/CmsGenericBlock'
+
+export default {
+  components: {
+    CmsGenericBlock,
+  },
+  name: 'CmsSectionSidebar',
+  props: {
+    content: {
+      type: Object,
+      default: () => ({}),
+    },
+  },
+  computed: {
+    getBlocks() {
+      return this.content.blocks || []
+    },
+    getSidebarBlocks() {
+      return this.getBlocks.filter(
+        (block) => block.sectionPosition === 'sidebar'
+      )
+    },
+    getMainBlocks() {
+      return this.getBlocks.filter((block) => block.sectionPosition === 'main')
```

Comment:
Code style, compare to line 39-41.

Reply:
prettier does that automatically :)
one line is clear to read,  but seems like in 39 it was too long